arviz/plots/khatplot.py

```diff
@@ -136,7 +138,7 @@ def plot_khat(
     hlines_kwargs.setdefault("linestyle", [":", "-.", "--", "-"])
     hlines_kwargs.setdefault("alpha", 0.7)
     hlines_kwargs.setdefault("zorder", -1)
-    hlines_kwargs.setdefault("color", "C1")
+    hlines_kwargs.setdefault("color", vectorized_to_hex("C1"))
```

Comment:
This will only transform `C1` to hex without transforming `hlines_kwargs["color"]` to hex if the users inputs a value for it. Try printing `hlines_kwargs` right below here and compare this approach to the other proposal if you do not pass any hlines_kwargs and if you do pass something like `["k", "k", "k", "r"]`